apps/sms/desktop-mock/navigator_moz_mobilemessage_shim.js

```diff
@@ -7,12 +7,12 @@
 
 *********************************************************** */
 (function(window) {
- 
-  if (!parent.navigator.mozMobileMessage) {
+
+  if (!opener || !opener.navigator.mozMobileMessage) {
```

Comment:
I think with little-browser, we don't have to care about this much :) This mock will be only in top level frame, and the rest should use it instead.